### PR TITLE
55 see test reports visualized

### PR DESCRIPTION
**Pull request by ToMer-K**
issue #55

### CODE REVIEW

File: .github/workflows/practica-quality-check.yml [hunk 1]
```diff
@@ -31,8 +31,12 @@ jobs:
 
     - run: npm ci
 
-    - name: Generate coverage report
-      run: npm test -- --coverage
+    - name: Run tests and generate coverage report
```
Comment (rluvaton) [inline .github/workflows/practica-quality-check.yml:34]: could you please disable the code coverage report as we already have codecov 😄 

for more how to do it - [`mattallty/jest-github-action` - Silencing the code coverage comment](https://github.com/mattallty/jest-github-action#silencing-the-code-coverage-comment)

**Review by rluvaton — COMMENTED** [review]
Great addition, thanks!

File: docs/docs/features-reference.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ Here will come a filter panel to search by categories, what's strategic, and mor
 ## 3. Testing experience
 ### 3.1 Slow tests detection
 
-**What:** Slow tests automatically shown clearly in the console and exported to csv + json reports
+**What:** Slow tests automatically shown clearly in the console and exported to a json report
```
Comment (goldbergyoni) [inline docs/docs/features-reference.md:61]: Great!

File: .github/workflows/practica-quality-check.yml (hunk 1 above)
Comment (goldbergyoni) [reply .github/workflows/practica-quality-check.yml:34]: Yea, this would result in abmiguity

**Review by goldbergyoni — APPROVED** [review]
@ToMer-K Awesome. Left minor comment, decide if/what to fix and merge yourself

File: .github/workflows/practica-quality-check.yml (hunk 1 above)
Comment (ToMer-K) [reply .github/workflows/practica-quality-check.yml]: I removed the coverage comment

**Comment by ToMer-K** [comment]
@goldbergyoni I didn't change the name of "Run tests and generate coverage report" because it does create the report.
In the next step it is used in codecov

**Pull request merged**